docs/team/drwala.md

```diff
@@ -1,53 +1,98 @@
+
```

Comment:
Probably should remove this blank line since it is making the meta info rendered in the page

Reply:
Fixed